temporal-sdk/src/main/java/io/temporal/internal/worker/WorkflowWorker.java

```diff
@@ -304,6 +304,7 @@ public void handle(WorkflowTask task) throws Exception {
       MDC.put(LoggerTag.WORKFLOW_ID, workflowExecution.getWorkflowId());
       MDC.put(LoggerTag.WORKFLOW_TYPE, workflowType);
       MDC.put(LoggerTag.RUN_ID, runId);
+      MDC.put(LoggerTag.ATTEMPT, Integer.toString(workflowTaskResponse.getAttempt()));
```

Comment:
We should not include this tag on the workflow task attempt for `ATTEMPT` as it has a different meaning then the ATTEMPT for local or normal activities

Reply:
Ok, I will remove it. But what's the meaning of attempt when dealing with workflows? Do you have any documentation regarding this topic?

Reply:
attempt here is the workflow task attempt not the attempt of the workflow

Reply:
in case of replay, that attempt will increase? Or it is related more to retries, like on Activities?

Reply:
Yes, it is not like retries on activities